lms/djangoapps/commerce/signals.py

```diff
@@ -140,33 +141,73 @@ def refund_seat(course_enrollment, request_user):
     return refund_ids
 
 
+def create_zendesk_ticket(requester_name, requester_email, subject, body, tags=None):
+    """ Create a Zendesk ticket via API. """
+    if not (settings.ZENDESK_URL and settings.ZENDESK_USER and settings.ZENDESK_API_KEY):
+        log.debug('Zendesk is not configured. Cannot create a ticket.')
+        return
+
+    # Copy the tags to avoid modifying the original list.
+    tags = list(tags or [])
+    tags.append('LMS')
+
+    # Remove duplicates
+    tags = list(set(tags))
+
+    data = {
+        'ticket': {
+            'requester': {
+                'name': requester_name,
+                'email': requester_email
+            },
+            'subject': subject,
+            'comment': {'body': body},
+            'tags': tags
+        }
+    }
+
+    # Encode the data to create a JSON payload
+    payload = json.dumps(data)
+
+    # Set the request parameters
+    url = urljoin(settings.ZENDESK_URL, '/api/v2/tickets.json')
+    user = '{}/token'.format(settings.ZENDESK_USER)
+    pwd = settings.ZENDESK_API_KEY
+    headers = {'content-type': 'application/json'}
+
+    response = requests.post(url, data=payload, auth=(user, pwd), headers=headers)
```

Comment:
I'd wrap this in an exception handler to catch timeouts and other issues that might occur when making the request. We'd like to know when Zendesk tickets can't be created, but we wouldn't want that to break the rest of our code.